content/applications/hr/payroll.rst

```diff
@@ -507,7 +677,7 @@ Model`, :guilabel:`Related Field`, and :guilabel:`Category`. Select a :guilabel:
 the drop-down menu. :guilabel:`Employee` populates the field by default, but the :guilabel:`Bank
```

Comment:
please break-up the information/sentences from 675-682 into more digestible lines

Reply:
I lost track of the lines because I added SO MAY lines to this PR =D. I believe I have chopped everything up and did a quick scan of the doc to make sure nothing was too bulky in each paragraph, so I think this was addressed.